woocommerce/woocommerce
# Added Ukrainian states

https://en.wikipedia.org/wiki/Oblasts_of_Ukraine

### All Submissions:


### Changes proposed in this Pull Request:

Added new Ukrainian states.

### How to test the changes in this Pull Request:

1. Apply this change and run `composer install`.

2. Place a new order. In the checkout field select Ukraine as the country, verify that you can select one of the existing states in the region dropdown.

3. Go to the admin area, open the order details, verify that the country and region order is correct and you can change the region in the region dropdown.

### Other information:


### Changelog entry

>Enhancement - Added new Ukrainian states.
